src/test/java/uk/gov/pay/ledger/transaction/resource/TransactionResourceIT.java

```diff
@@ -686,9 +686,10 @@ public void shouldGetAllTransactionsAsCSVWithAcceptType() throws IOException {
                 .withDefaultTransactionDetails()
                 .insert(rule.getJdbi());
 
+        // @TODO(sfount) parameterised tests if both methods will be supported long term
         InputStream csvResponseStream = given().port(port)
                 .accept("text/csv")
-                .get("/v1/transaction?" +
+                .get("/v1/transaction/stream?" +
```

Comment:
hmm this leaves the current route untested for any regressions even for shorter period